contrib/vagrant/start.sh

```diff
@@ -355,7 +356,8 @@ if [ -n "\${K8S}" ]; then
     done
 fi
 
-service cilium restart
+systemctl restart cilium-health
+systemctl restart cilium
```

Comment:
Hmm, since the systemd unit for cilium-health relies on the cilium unit, would restarting cilium restart cilium health too? I don't know what systemd does with it's dependency graph.

Reply:
It should, I will check and let you know later. And of course correct the docs.